conference.js

```diff
@@ -75,6 +75,8 @@ import {
     getAvatarURLByParticipantId,
     getLocalParticipant,
     getParticipantById,
+    hiddenParticipantJoined,
```

Comment:
Drive-by observation: we should not need these. A participant can have a hidden property if that's needed. Or alternatively we could filter transcribers before dispatching the participantJoined action.

Reply:
I disagree; currently a PARTICIPANT_JOINED event triggers whatever logic is required for a new participant, such as displaying it in the filmstrip. All this code should then first check if it's not actually a hidden participant?